docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  fdk-model-publisher:
+    build: .
+    image: eu.gcr.io/digdir-fdk-infra/fdk-model-publisher:test
+    ports:
+      - "8080:8080"
```

Comment:
Spesifiser gjerne alle miljøvariabler som skal brukes med tilsvarende verdier for staging miljø

Reply:
atm har den ikke noen miljøvariabler, legger de til etterhvert som de opprettes